lnd_services.go

```diff
@@ -766,3 +766,59 @@ func getClientConn(cfg *LndServicesConfig) (*grpc.ClientConn, error) {
 
 	return conn, nil
 }
+
+// GetTLSCredentials gets the tls credentials, whether provided as straight-up
+// data or a path to a certificate file.
+func GetTLSCredentials(tlsData, tlsPath string) (
```

Comment:
Any reason why this is an exported function?

Reply:
Indeed, it comes in handy in this PR: https://github.com/lightninglabs/lightning-terminal/pull/245